tests/data/not-autoloaded/composer/sample.json

```diff
@@ -12,6 +12,17 @@
         ],
         "files": [
             "dir2/file1.php"
+        ],
+        "exclude-from-classmap": [
+            "/dir2///*.php",
+            "/dir3/**/file1.php",
```

Comment:
I believe we need a test that ensures the generated regex is working and actually ignores expected files.

Reply:
Not exactly sure how it should be, but added a few more tests https://github.com/shipmonk-rnd/composer-dependency-analyser/pull/140/commits/2b20aef2472fe2a96c9c1de785ce82309f7611b0